pkg/testutil/testutil.go

```diff
@@ -79,6 +79,21 @@ func Equals(tb testing.TB, exp, act interface{}, v ...interface{}) {
 	tb.Fatal(sprintfWithLimit("\033[31m%s:%d:"+msg+"\n\n\texp: %#v\n\n\tgot: %#v%s\033[39m\n\n", filepath.Base(file), line, exp, act, diff(exp, act)))
 }
 
+// NotEqual fails the test if exp is equal to act.
+func NotEqual(tb testing.TB, exp, act interface{}, v ...interface{}) {
```

Comment:
I would be strict and suggest removing it. It would be much better to assert on the expected thing, then assert that thing is not value X ...  WDYT? (:

Reply:
Yeah fair enough - adding this did feel a bit sketchy :sweat_smile: